src/source.css

```diff
@@ -9670,11 +9765,72 @@ img[src="/assets/616e078b351d0df460971441949c53a3.svg"] {
   background-color: var(--hover-color) !important;
 }
 
+/* Community settings */
+.upsellContainer-L9xv7w,
+.backgroundAccent-349kuI,
+.checklistContainer-mFJZEJ,
+.checklistHeader-1KWcEY,
+.checklist-3Y6Fqp,
+.header-2Y0-A-,
+.enableContainer-6E-puu,
+.previewContainer-1SS3uO,
+.welcomeChannel-1rFrIO {
+  background-color: rgba(0, 0, 0, 0.4) !important;
+  color: var(--hover-bright-color) !important;
+}
+.developerPortalCtaWrapper-2XNafh {
+  background-color: var(--hover-color) !important;
+}
+.developerPortalCtaWrapper-2XNafh,
+.text-1ebFA_,
+.base-1x0h_U {
+  color: var(--hover-bright-color) !important;
+}
+.upsellFooter-ZYsio_ {
+  background-color: transparent !important;
+}
+
 /* Server Template */
 .descriptionBox-1EKQKL {
   background-color: rgba(0, 0, 0, 0.2) !important;
 }
 
+/* Server Template Preview */
+.ctaSection-izWwhs,
+.formSection-1NFAGI {
+  background-color: rgba(0, 0, 0, 0.4) !important;
+}
+.channelsWrapper-2HhUER,
+.rolesWrapper-2yOx9S {
+  background-color: transparent !important;
+  border: 1px solid var(--hover-color) !important;
+}
+.usagePill-_nSrnP {
+  background-color: var(--hover-color) !important;
+  color: var(--hover-bright-color) !important;
+}
+.icon-3U3Ats {
+  color: var(--hover-bright-color) !important;
+}
+.icon-3U3Ats > div > svg > circle {
+  fill: var(--hover-bright-color) !important;
+}
+
+/* Welcome Screen */
+.optionContainer-15srkc {
+  background-color: transparent !important;
+  border: 1px solid var(--hover-color) !important;
+}
+.header-QjU-Vz {
+  color: var(--hover-bright-color) !important;
+}
+
```

Comment:
Additional coloring for "Welcome Screen" setting category of a freshly made server
```css
.exampleContainer-25sB-A {
  background-color: var(--hover-dark-color) !important;
}
.exampleModal-2X2Vf8 {
  background-color: rgba(0, 0, 0, 0.5) !important;
}
.optionContainer-1FtykV {
  background: var(--hover-color) !important;
}
.exampleModal-2X2Vf8 > h3 > strong {
  color: var(--soft-bright-color) !important;
}
.tooltip-1_vJJI {
  -webkit-box-shadow: none !important;
  box-shadow: none !important;
  background-color: var(--main-color) !important;
  color: var(--soft-bright-color) !important;
}
.tooltipPointer-1awMxk {
  border-right-color: var(--main-color) !important;
}
```